_docs/data/data.yaml

```diff
@@ -293,3 +293,36 @@ properties:
       This should be used with caution and avoided whenever possible.
     type: list
     required: false
+
+  - name: registries
+    description: |
+      Credentials for registries.
+      Example:
```

Comment:
I would not include the example twice. If we add it to the `Examples` section already, that should be enough.

Reply:
7f7cff265e2d05004209aedcbd848a8ac19fde1a